### ISSUE TITLE
Traffic island quest cannot be finished

### ISSUE THREAD

**Issue by DennisKempel** [streetcomplete/StreetComplete#5227]
**How to Reproduce**

**Expected Behavior**

**Versions affected**

**Comment by DennisKempel** [comment]
![Screenshot_20230901-055902](https://github.com/streetcomplete/StreetComplete/assets/65849470/cd7385c1-6534-465a-8c8c-6bd721522f9c)

**Comment by DennisKempel** [comment]
checkmark button lower right stays grey, no matter what I do

**Comment by qugebert** [comment]
The [node](https://www.openstreetmap.org/node/8869354461#map=19/53.55854/9.91370&layers=N) is already tagged as traffic_island, so your selection wouldn't change anything. Maybe this is the reason why the button is disabled.

For those who wonder how to find the quest: It is not a quest, but the StreetParking Overlay.

**Comment by westnordost** [comment]
Yes, this is the reason. You can only apply the change if you made a change ([for the time being](https://github.com/streetcomplete/StreetComplete/issues/4735)).
